### PR TITLE
Demonstrator tweaks

### PR DESCRIPTION
**Pull request by VictorLamoine**
- Update defect mesh
- Add trajectory Z offset in path planning
- Set simulation initial robot start pose
- Tweak TP program name (add `.ls` suffix)
- Update RViz config

### CODE REVIEW

**Comment by VictorLamoine** [comment]
@KevinBollore ready for review

File: path_planning/src/path_planning.cpp [hunk 1]
```diff
@@ -83,6 +81,7 @@ bool pathPlanning(fanuc_grinding_path_planning::PathPlanningService::Request &re
     BezierGrindingSurfacing bezier(input_mesh_filename, req.GrinderWidth, req.CoveringPercentage, req.ExtricationRadius,
                                    req.LeanAngle, lean_axis);
 
+
```
Comment (KevinBollore) [inline path_planning/src/path_planning.cpp]: Remove this.

**Comment by KevinBollore** [comment]
Is there fondamental changes in your robot_state_visualize file?
If not, you can delete it.

**Comment by VictorLamoine** [comment]
Yes; the grinding parameters that we used for the demonstration

**Pull request merged**